jaeger/README.md

```diff
@@ -0,0 +1,180 @@
+- kubectl >= 1.22.x
+- Helm 3.x
+
+## Installation
+
+### Preparation
+
+1. Export your Namespace as a variable. Replace the `{namespace}` placeholder in the following command and run it:
+
+    ```bash
+    export KYMA_NS="{namespace}"
+    ```
+
+1. Export the Helm release name that you want to use. The release name must be unique for the chosen Namespace. Be aware that all resources in the cluster will be prefixed with that name. Run the following command:
+    ```bash
+    export HELM_JAEGER_RELEASE="jaeger"
+    ```
+
+1. Update your Helm installation with the required Helm repository:
+
+    ```bash
+    helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
+    helm repo update
+    ```
+
+### Install Jaeger
+
+> **NOTE:** Jaeger recommends to install Jaeger with the [Jaeger operator](https://github.com/jaegertracing/helm-charts/tree/main/charts/jaeger-operator); As the operator requires a cert-manager to be installed, the instructions here will use a plain Jaeger installation. Anyway the installation described is not meant to be used for productive setups.
+
+Run the Helm upgrade command, which installs the chart if not present yet.
+```bash
+helm upgrade --install --create-namespace -n $KYMA_NS $HELM_JAEGER_RELEASE jaegertracing/jaeger -f https://raw.githubusercontent.com/kyma-project/examples/main/jaeger/values.yaml
+```
+
+You can either use the [`values.yaml`](./values.yaml) provided in this `jaeger` folder, which contains customized settings deviating from the default settings, or create your own `values.yaml` file.
+
+### Verify the installation
+
+Check that the `jaeger` Pod has been created in the Namespace and is in the `Running` state:
+```bash
+kubectl -n $KYMA_NS rollout status deploy $HELM_JAEGER_RELEASE
+```
+
+### Activate a TracePipeline
+
+To configure the Kyma trace collector with the deployed Jaeger instance as backend, create a new [TracePipeline](https://kyma-project.io/docs/kyma/main/01-overview/main-areas/telemetry/telemetry-03-traces/) by 
+executing the following command.
+   ```bash
+   cat <<EOF | kubectl -n $KYMA_NS apply -f -
+   apiVersion: telemetry.kyma-project.io/v1alpha1
+   kind: TracePipeline
+   metadata:
+     name: jaeger
+   spec:
+     output:
+       otlp:
+         protocol: http
+         endpoint:
+           value: http://$HELM_JAEGER_RELEASE-collector.$KYMA_NS.svc.cluster.local:4318
+   EOF
+   ```
+  
+### Activate Istio Tracing
+
+To [activate Istio](https://kyma-project.io/docs/kyma/main/01-overview/main-areas/telemetry/telemetry-03-traces#step-2-enable-istio-tracing) to report span data, an Istio telemetry resource needs to get applied, setting the sampling rate to 100% (not recommended for production).
+
+```yaml
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: tracing-default
+  namespace: istio-system
+spec:
+  tracing:
+  - providers:
+    - name: "kyma-traces"
+  randomSamplingPercentage: 100.00
```

Comment:
The two spaces are missing. `randomSamplingPercentage ` should be on the same level as `providers `